.github/workflows/ci_e2e-unstable.yaml

```
@@ -33,15 +33,26 @@ jobs:
             DATABASE_URL: "mysql://root:root@127.0.0.1/sylius?charset=utf8mb4&serverVersion=${{ matrix.mysql }}"
 
         steps:
+            -   name: Set variables
+                shell: bash
+                env:
+                    BRANCH: ${{ inputs.branch }}
```

actionlint (1.7.7)

property "branch" is not defined in object type {ignore-failure: bool}

(expression)